lib/bloc/payment_options/payment_options_bloc.dart

```diff
@@ -11,7 +11,7 @@ class PaymentOptionsBloc extends Cubit<PaymentOptionsState> {
   PaymentOptionsBloc(
     this._preferences,
   ) : super(const PaymentOptionsState.initial()) {
-    getPaymentOptions();
+    resetPaymentOptions(true);
   }
```

Comment:
Realized we can simplify this whole logic to:

```suggestion
        _initializePaymentOptions();
  }

  void _initializePaymentOptions() async {
    await _preferences.hasPaymentOptions() ? await getPaymentOptions() : await resetPaymentOptions();
  }
```

Reply:
Looks a lot simpler now